migrations/20210220144200_organization_contacts.js

```diff
@@ -29,7 +29,7 @@ exports.up = async function up(knex) {
     t.integer("last_error_code");
     t.text("carrier");
     t.timestamp("created_at").defaultTo(knex.fn.now());
-    t.timestamp("last_lookup").defaultTo();
+    t.timestamp("last_lookup").defaultTo(null);
```

Comment:
do `null`s need to be added anywhere else in the migrations?

Reply:
Nope! This migration was the only one causing issues because it had a `.defaultTo()` with nothing inside the parentheses.